src/main/java/hudson/plugins/accurev/AccurevSCM.java

```diff
@@ -991,6 +995,10 @@ public DescriptorImpl getDescriptor() {
     @Extension
     public static class DescriptorImpl extends Descriptor<AccurevServer> {
 
+      public DescriptorImpl() {
```

Comment:
🙌 Would be nice if you could actually add a unit test for JCasc cause this change enables accurev servers to be managed from JCasc.
https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/PLUGINS.md#how-to-test-